samples/zigbee/light_bulb/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _zigbee_light_bulb_sample:
+
+Zigbee: Light bulb
+##################
+
+This Zigbee light bulb sample demonstrates a simple light bulb whose brightness can be regulated by a different device.
+
+You can use this sample together with :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` and Zigbee light switch to set up a basic Zigbee network.
+
+Overview
+********
+
+The Zigbee light bulb sample assumes the Zigbee Router role and implements the Dimmable Light profile.
+This profile allows to change the brightness level of a LED on the light bulb development kit.
```

Comment:
What is a light bulb development kit? ;) Probably the development kit that runs the light bulb sample?

Reply:
Fixed.